third_party/CMakeLists.txt

```diff
@@ -58,6 +58,15 @@ if(NOT USE_SHARED_TINYXML)
   add_subdirectory(tinyxml2)
 endif()
 
+# Jpeg exif fields
+if(NOT USE_SHARED_TINYEXIF)
+  set(TinyEXIF_BUILD_TESTING OFF CACHE BOOL "Build tests for tinyexif")
+  add_subdirectory(TinyEXIF)
```

Comment:
Here the subdirectory is added as "TinyEXIF" but the submodule cloned in "third_party/tinyexif". You should clone add the submodule as it is: "TinyEXIF" in "third_party/TinyEXIF", rename the repository etc. Probably you will need to remove the submodule and add again correctly in this commit.

Exactly as what is done with IXWebSocket and https://github.com/aseprite/IXWebSocket